build/release.mk

```diff
@@ -190,6 +191,17 @@ else
 	done
 endif
 
+	@# Products
+
+	@if [ -d $(BUILD_BOARDS_DIR) ] ; then \
+		echo "Copying web app files for Boards product"; \
+		mkdir -p $(DIST_PATH_GENERIC)/client/products/boards; \
+		cp -R $(BUILD_BOARDS_DIR)/mattermost-plugin/webapp/dist/* $(DIST_PATH_GENERIC)/client/products/boards/; \
+		ls $(DIST_PATH_GENERIC)/client/products/boards; \
```

Comment:
Any need to run `ls`?

Reply:
It's not strictly necessary. I left it there since I thought it might be helpful, but I can remove it